src/ports/postgres/modules/graph/test/wcc.sql_in

```diff
@@ -0,0 +1,92 @@
+/* ----------------------------------------------------------------------- *//**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ *//* ----------------------------------------------------------------------- */
+
+DROP TABLE IF EXISTS vertex, edge;
+CREATE TABLE vertex(
+vertex_id INTEGER
+);
+CREATE TABLE edge(
+src_node INTEGER,
+dest_node INTEGER,
+user_id INTEGER
+);
+INSERT INTO vertex VALUES
+(0),
+(1),
+(2),
+(3),
+(4),
```

Comment:
When there is no grouping, a node that has no edge must to be considered a separate component. 

In the current grouping implementation of other graph modules such as PageRank and SSSP, if a node has no edge, it is not associated with any group. The current implementation in this PR (if extended to support grouping) is consistent with that assumption.

If we want to capture a disconnected node as part of a particular group, we will have to discuss and see how to represent it in the edge table. That discussion should be in a new thread as it is beyond the scope of this PR. So I am assuming a no-op on this comment.